module/data/item/armor.mjs

```diff
@@ -3,102 +3,108 @@ import ItemDescriptionTemplate from "./templates/item-description.mjs";
-                } ),
-                forgeBonus: new fields.NumberField( {
-                    required: true,
-                    nullable: false,
-                    min: 0,
-                    initial: 0,
-                    integer: true,
-                    label: "ED.Item.Armor.forgeBonusMystical"
-                } ),
-            } ),
-            initiativePenalty: new fields.NumberField( {
-                required: true,
-                nullable: false,
-                min: 0,
-                initial: 0,
-                integer: true,
-                label: "ED.Item.Armor.initiativePenalty"
-            } ),
-            piecemealArmor: new fields.SchemaField( {
-                selector: new fields.BooleanField( {
-                    required: true,
-                    initial: false,
-                    label: "ED.Item.Armor.piecemealArmor"
-                } ),
-                size: new fields.NumberField( {
-                    required: true,
-                    nullable: false,
-                    min: 0,
-                    max: 3,
-                    initial: 0,
-                    integer: true,
-                    label: "ED.Item.Armor.piecemealArmorSize"
-                } ),
-            }, {
-                required: true,
-                nullable: false,
-                label: "ED.Data.Item.Labels.piecemealArmor",
-                hint: "ED.Data.Item.Hints.piecemealArmor",
-            } ),
-        } );
-    }
+  /** @inheritDoc */
+  static defineSchema() {
+    const fields = foundry.data.fields;
+    return this.mergeSchema( super.defineSchema(), {
+      physical: new fields.SchemaField( {
+        armor: new fields.NumberField( {
+          required: true,
+          nullable: false,
+          min:      0,
+          initial:  0,
+          integer:  true,
+          label:    "ED.Item.Armor.physicalArmor"
+        } ), 
+        forgeBonus: new fields.NumberField( {
+          required: true,
+          nullable: false,
+          min:      0,
+          initial:  0,
+          integer:  true,
+          label:    "ED.Item.Armor.forgeBonusPhysical"
+        } ),
+      } ),
+      mystical: new fields.SchemaField( {
+        armor: new fields.NumberField( {
+          required: true,
+          nullable: false,
+          min:      0,
+          initial:  0,
+          integer:  true,
+          label:    "ED.Item.Armor.mysticalArmor"
+        } ),
+        forgeBonus: new fields.NumberField( {
+          required: true,
+          nullable: false,
+          min:      0,
+          initial:  0,
+          integer:  true,
+          label:    "ED.Item.Armor.forgeBonusMystical"
+        } ),
+      } ),
+      initiativePenalty: new fields.NumberField( {
+        required: true,
+        nullable: false,
+        min:      0,
+        initial:  0,
+        integer:  true,
+        label:    "ED.Item.Armor.initiativePenalty"
+      } ),
+      isLivingArmor: new fields.BooleanField( {
```

Comment:
Sorry, yet another suggestion 😅 could we just name it `living`? the item already is armor, so we don't need to repeat item types in properties (this should also apply to things like "piecemealArmor" actually..)
It's always difficult to find a good compromise between descriptive property names and well legible code... But I would suggest that in general: properties don't need information about the object their part of in it...

Reply:
perfect!